src/pages/thunderbird/faq.js

```diff
@@ -0,0 +1,173 @@
+import React  from 'react';
+import MozillaFooter from '../../components/mozilla/footer.js';
+import Link from '../../components/link.js';
+var FormattedHTMLMessage = require("react-intl").FormattedHTMLMessage;
+
+var Panel = React.createClass({
+  onClick: function() {
+    var itemKey = this.props.itemKey;
+    if (itemKey === this.props.activeKey) {
+      itemKey = "";
+    }
+    this.props.activateKey(itemKey);
+  },
+  render: function() {
+    var itemClassName = "collapse-item";
+    var contentClassName = "collapse-content";
+    if (this.props.itemKey === this.props.activeKey) {
+      itemClassName += " collapse-item-active";
+      contentClassName += " collapse-content-active";
+    }
+    return (
+      <div id={this.props.itemKey} className={itemClassName}>
+        <div onClick={this.onClick} className="collapse-header">{this.props.header}</div>
+        <div className={contentClassName}>
+          {this.props.children}
+        </div>
+      </div>
+    );
+  }
+});
+
+var Faq = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
+  getInitialState: function() {
+    return {
+      activeKey: ""
+    };
+  },
+  updateKey: function(newActiveKey) {
+    if (newActiveKey !== this.state.activeKey) {
+      this.setState({
+        activeKey: newActiveKey
+      });
+    }
+  },
+  updateHash: function(key) {
+    var hash = "";
+    if (key) {
+      hash = "#" + key;
+    }
+    history.replaceState({}, '', window.location.origin + window.location.pathname + window.location.search + hash);
+    this.updateKey(key);
+  },
+  componentDidUpdate: function() {
+    this.updateKey(window.location.hash.replace("#", ""));
+  },
+  componentDidMount: function() {
+    this.updateKey(window.location.hash.replace("#", ""));
+  },
+  onKeyChange: function(key) {
+    this.updateHash(key);
+  },
+  renderEnglishTaxItem: function() {
+    if (/^(en)(\b|$)/.test(this.context.intl.locale)) {
+      return (
+        <Panel activeKey={this.state.activeKey} activateKey={this.onKeyChange} itemKey="item_tax_b" header="How can I get a receipt for U.S. tax preparation purposes?">
+          <p>If you give to Thunderbird through an online form, you should immediately receive an emailed receipt for your donation that you can use for your U.S. tax preparation purposes. Check your spam folder if you don’t see that email arrive in your email inbox, as it can sometimes be caught by spam filters. If you need an additional copy of your receipt for you for your U.S. tax preparation purposes, please contact us at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.</p>
```

Comment:
Is the email address correct here? I know you’re using donate@ it in some cases, you want to make sure it’s not a mistake

Reply:
yes, this is fine.